src/content/blog/astro-470.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Astro 4.7"
+description: "Astro 4.7 is out now! This release includes major improvements to the API for making toolbar apps, a new way to keep yourself up to date, and more."
+publishDate: "April 25, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
```

Comment:
```suggestion
  - bjorn
  - sarah
```

lol, I'm (half) joking, but I feel pretty invested and involved in these releases! :smile:

Reply:
There's an infinite amount of room for this, get in!